exporters/otlp/trace/build.gradle.kts

```diff
@@ -29,8 +29,6 @@ dependencies {
     exclude(mapOf("module" to "proto"))
```

Comment:
I noticed this produces a weird POM file - I promise to fix by the 1.6 release :P

Reply:
Fixed in #3520